app-text/xapers/xapers-0.9.0.ebuild

```diff
@@ -9,8 +9,8 @@ DISTUTILS_USE_SETUPTOOLS=rdepend
 inherit distutils-r1
 
```

Comment:
Any chance of 3.9?

Reply:
Needs `dev-libs/xapian-bindings[${PYTHON_USEDEP}]` but that's still only on 3.8. I mean, technically I could "enable" it, but I cannot give any promises as to whether it would function at all, let alone whether tests would pass. Also, means bumping pybtex which also does not have python 3.9, but at least I proxy-maintain that.

Reply:
That's ok, we are just trying to get 3.9 where possible. This PR is on my queue to test, thank you!